changelog/README.md

```diff
@@ -21,6 +21,16 @@ under the License.
 
 <!-- Note: contrary to 3.x, insert new entries *first* in their section -->
 
+### 4.18.0
+
+- [improvement] PR 1689: Add support for publishing percentile time series for the histogram metrics (nparaddi-walmart)
```

Comment:
Historically we've only included JAVA tickets in the changelog but this will obviously be changing as we transition to CASSANDRA tickets for the Java driver.  Rather than creating JAVA tickets for a few user-contributed changes (such as this one) it seemed easier to just include an explicit reference to the PR.

Reply:
I like including the CASSANDRA jira if there is one, I think the particular cases where there is no jira, referencing the PR is good (which I think was both of these) :+1:

At some point will be push for all contributions to be tied to a jira ticket?

Reply:
Yeah, both of the commits in question for this change came in as contributions from the community and weren't tied to any existing ticket (either JAVA or CASSANDRA).

Honestly I'm kind of in two minds about the idea of connecting everything to a CASSANDRA ticket @tolbertam.  We tried to do something like that for a while on the Java side and I'm not sure there's a huge benefit.  The PR gives you an identifier/address/URL you can use to refer to the change and any discussion can happen on the PR itself... in a lot of ways it's similar to having a JIRA ticket.

I don't have super-strong feelings on the matter so I'm open to pursuing a policy of tying any change to a CASSANDRA ticket... but it's not immediately clear to me it offers the benefits it appears to.